doc/usecases/polyculture/review_plant_relationships.md

```diff
@@ -0,0 +1,28 @@
+# Use Case: Review Plant Relationships
+
+## Summary
+
+- **Scope:** Plants Layer
+- **Level:** User Goal
+- **Actors:** App User, App User
+- **Brief:** The quality of a user created plant relationship is assessed by others.
+- **Status:** Assigned
+- **Assignee:** Benjamin
+
+## Scenarios
+
+- **Precondition:**
+  - The user is logged into the app.
+  - She is looking at a [plant relationship created by another user](add_plant_relationships.md).
+- **Main success scenario:**
+  - The user comments on the relationship to note something that seems relevant to her.
+  - The user approves the relationship because she agrees with it.
+- **Alternative scenario:**
+  - The user only comments on the relationship but doesn't approve it and might suggest a different relationship (e.g. neutral). 
+  - No changes are made to the relationships confidence score.
+  - One administrator decides on the outcome.
+- **Error scenario:**
```

Comment:
I think there is a lot that can go wrong when we give the user the ability to write reviews:
The user submits an invalid comment or review e.g. special characters, empty input, SQL + XSS injection.

Reply:
I don't see a problem with special characters as our postgres db uses utf8 encoding.
SQL and XSS injections should be covered by prepared statements from diesel and sanitizing by react respectively.
Of course one could use those in a wrong way but I think the code review process will catch this.